Signal/src/ViewControllers/HomeView/Chat List/ChatListViewController+Actions.swift

```diff
@@ -21,7 +21,6 @@ extension ChatListViewController {
         if let previousIndexPath = renderState.indexPath(beforeThread: currentThread),
            let thread = self.thread(forIndexPath: previousIndexPath) {
             self.present(thread, action: .compose, animated: true)
-            tableView.selectRow(at: previousIndexPath, animated: true, scrollPosition: .none)
```

Comment:
There were some existing paths that would update the table selection when presenting a thread. Now that all presentations will automatically update the table selection, this is no longer necessary.